tests/e2e/mock_api/procedures/record.py

```diff
@@ -0,0 +1,18 @@
+import mgp
+import mgp_mock
+import test_utils
+
+
+@mgp.read_proc
+def compare_apis(ctx: mgp.ProcCtx) -> mgp.Record(results_dict=mgp.Map):
+    results = dict()
+
+    record = mgp.Record(a=1, b=2.0, c="3")
```

Comment:
bool?

Reply:
what about date types

Reply:
The point of this test is not to verify that the types work (all of them are either Python standard library or Python API, not the mock API), but to verify that the record created by the mock has the `fields` attribute.